pages/price-feeds/_meta.json

```diff
@@ -42,7 +42,7 @@
   "api-reference": "API Reference",
   "price-feeds": "Price Feeds",
   "current-fees": "Current Fees",
-  "sponsored-feeds": "Sponsored Feeds",
+  "pushed-feeds": "Push Feeds",
```

Comment:
What sounds good to you?
"Push Feeds" or "Pushed Feeds"?

Reply:
can you also rename the filename?